docs/src/pages/components/data-grid/rendering/rendering.md

```diff
@@ -258,15 +258,15 @@ You can disable column virtualization by setting the column buffer to a higher n
 
 As part of the customization API, the grid allows you to replace and override nested components with the `components` prop. The prop accepts an object with the following keys:
 
-- `header`: The component rendered above the column header bar.
+- `toolbar`: The component rendered above the column header bar.
```

Comment:
That's gone. You should rebase and update the api slots page

Reply:
Done, I've updated the API pages as well.